src/main/resources/application.properties

```diff
@@ -1,4 +1,6 @@
-spring.profiles.active=prod
+spring.profiles.active=file
 
 spring.mvc.pathmatch.matching-strategy = ANT_PATH_MATCHER
-spring.output.ansi.enabled=always
\ No newline at end of file
+spring.output.ansi.enabled=always
+spring.flyway.enabled=false
+spring.datasource.url=jdbc:postgresql://localhost:5432/invoicing
```

Comment:
To nie powinno być wam tutaj potrzebne.
Parametr ten jest używany tylko w przypadku bazy sql'owej.

Reply:
Bez tego nie wstaje nam aplikacja na profilu prod i dev.

Ten error nam wyskakuje, a w zadaniu:
Description:

Failed to configure a DataSource: 'url' attribute is not specified and no embedded datasource could be configured.

Reason: Failed to determine a suitable driver class


Action:

Consider the following:
	If you want an embedded database (H2, HSQL or Derby), please put it on the classpath.
	If you have database settings to be loaded from a particular profile you may need to activate it (the profiles memory are currently active).

W zadaniu była podpowiedź:
Try to implement the last line to solve that problem :)
We will work with the H2 database on the next lesson - for now please just add it to the classpath so Spring is satisfied.

Więc dodaliśmy tą linijkę żeby Spring był póki co usatysfakcjonowany.
Jak inaczej można to ogarnąć?

Reply:
Rozwiązaliśmy problem. Była to kwestia konfiguracji zależności w buildzie gradle'a.